packages/troika-three-text/src/GlyphsGeometry.js

```diff
@@ -112,6 +118,11 @@ class GlyphsGeometry extends InstancedBufferGeometry {
       0
     )
     sphere.radius = sphere.center.distanceTo(tempVec3.set(totalBounds[0], totalBounds[1], 0))
+
+    // Update the boundingBox based on the total bounds with an arbitrary depth of 20
```

Comment:
Thanks for the PR!

I'm curious, why an arbitrary depth of 20? Wouldn't a depth of 0 be correct, since it's a plane with no thickness? I think that would be consistent with e.g. PlaneBufferGeometry's boundingBox.

Also a nitpick, could you please change to mutating the min/max rather than replacing with newly allocated Vector3s? `box.min.set(...)` -- unless there's a reason for using new Vector3s.

Reply:
You're very welcome, thanks for the awesome library!

Depth of 0 makes sense, and so does mutation!